cpp/velox/CMakeLists.txt

```diff
@@ -91,9 +91,13 @@ macro(ADD_VELOX_DEPENDENCIES)
   add_velox_dependency(dwio::dwrf::common "${VELOX_COMPONENTS_PATH}/dwio/dwrf/common/libvelox_dwio_dwrf_common.a")
   add_velox_dependency(parquet "${VELOX_COMPONENTS_PATH}/dwio/parquet/libvelox_dwio_parquet_reader.a")
   add_velox_dependency(parquet::reader::duckdb "${VELOX_COMPONENTS_PATH}/dwio/parquet/duckdb_reader/libvelox_dwio_duckdb_parquet_reader.a")
-  add_velox_dependency(parquet::duckdb::reader "${VELOX_COMPONENTS_PATH}/dwio/parquet/duckdb_reader/duckdb/libvelox_dwio_duckdb_parquet_reader_duckdb.a")
+  add_velox_dependency(parquet::reader::native "${VELOX_COMPONENTS_PATH}/dwio/parquet/reader/libvelox_dwio_native_parquet_reader.a")
+  add_velox_dependency(parquet::reader::thrift "${VELOX_COMPONENTS_PATH}/dwio/parquet/thrift/libvelox_dwio_parquet_thrift.a")
+  add_velox_dependency(duckdb::duckdb "${VELOX_COMPONENTS_PATH}/external/duckdb/libduckdb.a")
+  add_velox_dependency(parquet::thrift "${VELOX_BUILD_PATH}/third_party/arrow_ep/src/arrow_ep-build/thrift_ep-install/lib/libthrift.a")  
```

Comment:
Looks like this lib is just from third-party, and is not connected with parquet. How about change the name from `parquet::thrift` to `thrift`?